packages/data/src/namespace-store/test/index.js

```diff
@@ -80,4 +80,59 @@ describe( 'controls', () => {
 
 		registry.select( 'store' ).getItems();
 	} );
+	describe( 'various action types have expected response and resolve as ' +
+		'expected', () => {
+		const actions = {
+			*withPromise() {
```

Comment:
Aside: We should devise a spacing convention for `*`, as it's not enforced one way or the other currently, and I've personally written code with the space between the `*` and the function name.

Not evident here, the guideline should include whether a space should occur _prior_ the `*` as in `function* getFoo` or `function * getFoo`

Reply:
My preference if we enforce some convention would be to do:

- `*withPromise() {` for functions in an object property shorthand
- ` function* getFoo` for declared functions.

Reply:
Related: The finishing touches are being put on `wp-pretter@2` which will answer all your `function * () {}` questions 😁